source/jsonnet/england-wales/ccs/blocks/who-lives-here/any_visitors.jsonnet

```
@@ -1,77 +1,116 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local otherHouseholdAddress = {
+  placeholder: 'other_address',
+  transforms: [
+    {
+      transform: 'concatenate_list',
+      arguments: {
+        list_to_concatenate: {
+          source: 'answers',
+          identifier: ['usual-address-answer-building', 'usual-address-answer-street'],
+        },
+        delimiter: ', ',
+      },
+    },
+  ],
+};
+
+local questionTitle(livesAtHouseholdAddress) = (
+  if livesAtHouseholdAddress then {
+    text: 'How many visitors were staying overnight at {household_address} on Sunday {census_date}?',
+    placeholders: [
+      placeholders.address,
+      placeholders.censusDate,
+    ],
+  }
+  else {
+    text: 'How many visitors were staying overnight at {other_address} on Sunday {census_date}?',
+    placeholders: [
+      otherHouseholdAddress,
+      placeholders.censusDate,
+    ],
+  }
+);
```

No need for the extra variable, this can just be included inline:
```suggestion
local questionTitle(livesAtHouseholdAddress) = (
  if livesAtHouseholdAddress then {
    text: 'How many visitors were staying overnight at {household_address} on Sunday {census_date}?',
    placeholders: [
      placeholders.address,
      placeholders.censusDate,
    ],
  }
  else {
    text: 'How many visitors were staying overnight at {other_address} on Sunday {census_date}?',
    placeholders: [
      {
        placeholder: 'other_address',
        transforms: [
          {
            transform: 'concatenate_list',
            arguments: {
              list_to_concatenate: {
                source: 'answers',
                identifier: ['usual-address-answer-building', 'usual-address-answer-street'],
              },
              delimiter: ', ',
            },
          },
        ],
      },
      placeholders.censusDate,
    ],
  }
);
```